File: Domain/Category.cs
using System;
using System.Collections.Generic;

namespace Domain
{
    public class Category
    {
        public Guid Id { get; set; }
        public string Code { get; set; }
        public string Image { get; set; }
        public string Name { get; set; }
        public string Color { get; set; }
        public DateTime CreationDate { get; set; }
    }
}

File: client-app/src/app/models/category.ts
export interface ICategory {
    id: string;
    code: string;
    image: string;
    name: string;
    color: string;
    creationDate: Date;
}

export interface ICategoryForm {
    code: string;
    image: Blob | string;
    name: string;
    color: string;
}

export class ICategoryFormValues implements ICategoryForm {
    name = "";
    code = "";
    image = new Blob();
    color = "";
}